osu.Game.Tests/Visual/UserInterface/TestSceneModSettings.cs

```diff
@@ -121,9 +141,21 @@ private class TestModSelectOverlay : ModSelectOverlay
 
             public bool ButtonsLoaded => ModSectionsContainer.Children.All(c => c.ModIconsLoaded);
 
+            public ModButton GetModButton(Mod mod)
+            {
+                return ModSectionsContainer.Children.Single(s => s.ModType == mod.Type)
```

Comment:
Going through `Children` and `ButtonsContainer` can probably be skipped. `ChildrenOfType<>` works recursively across the entire subtree.